tensorflow/core/platform/default/logging.h

```diff
@@ -71,8 +72,8 @@ class LogMessageFatal : public LogMessage {
 
 #define LOG(severity) _TF_LOG_##severity
 
-// TODO(jeff): Define a proper implementation of VLOG_IS_ON
-#define VLOG_IS_ON(lvl) ((lvl) <= 0)
+// Set TF_CPP_MIN_VLOG_LEVEL environment to update minimum log level of VLOG
```

Comment:
for binary size reasons, I think it's preferable on mobile devices to not include vlog support.  So can you include somthing like:

#ifdef IS_MOBILE_PLATFORM
/// old VLOG_IS_ON code
#else
// new VLOG_IS_ON code
#endif